Packs/HYASInsight/Integrations/HYASInsight/README.md

```diff
@@ -291,128 +299,10 @@ Returns WHOIS records for the provided indicator value.
 #### Command Example
 ```!hyas-get-whois-records-by-indicator indicator_type="domain" indicator_value="domain.net" limit="3"```
```

Comment:
I'm not sure why the context example and Human Readable Output of 'hyas-get-whois-records-by-indicator' command was removed. was it on purpose?